static/styles/recent_topics.css

```diff
@@ -214,6 +214,12 @@
             }
         }
 
+        tbody {
+            tr {
+                cursor: pointer;
```

Comment:
@amanagr can you review?  I'm a bit skeptical of this not going in the `tr` block just above or just the topic row block if that's how the click handler works.

Reply:
This change should only happen for topic and stream column, not the entire row.

Reply:
@amanagr i think the issue here is that when we hover over the rows, the background color changes in response to the hover and is also clickable. Since, it it clickable and has a response to hover, it would be better for cursor to be pointer.

Reply:
The area near buttons and participants is not clickable in the row.